### PR TITLE
Development

### PR DESCRIPTION
**Pull request by sbmadhav**
Fix react-styleguidist

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
```
Comment (vish288) [inline docs/index.html]: could we gitignore the docs and ``git rm `` them from the repo,if these are generated via styleguide pls

File: fusion/Avatar.js [hunk 2]
```diff
@@ -15,13 +15,13 @@ const Avatar = ({ src, alt }) => (
 
 Avatar.propTypes = {
   src: PropTypes.string,
-  alt: PropTypes.string,
+  alt: PropTypes.string
 };
```
Comment (vish288) [inline fusion/Avatar.js]: i think a . bunch of these are from prettier in husky.. is there a reason we have both npm run format and npm run prettier?

File: package.json [hunk 3]
```diff
@@ -20,9 +20,15 @@
...
     "react-live": "^1.9.2",
     "rimraf": "^2.6.2",
+    "stylis": "^3.4.10",
     "theming": "^1.3.0"
   },
   "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (vish288) [inline package.json]: we should soon move to present-env.

**Review by vish288 — APPROVED** [review]

**Comment by vish288** [comment]
@sbmadhav : there are some recent changes pushed from my side to clean up few things. please take a note of those and we can merge.

**Comment by sbmadhav** [comment]
@vichu1988 - the docs are required to generate github pages

**Pull request merged**